pkg/execution/execute.go

```diff
@@ -384,9 +384,16 @@ func runTestRule(rule windup.When, violations []hubapi.RuleSet) (bool, int, int)
 
 	numFound := 0
 	var tags []string
+
+	r := strings.NewReplacer(
```

Comment:
what if we just compiled the regex strings and compared them instead of doing the textual comparisons? Might net us a few more passes if there are other more complex regex patterns in there

Reply:
yeah that's pretty much exactly what I was thinking